src/utils/dotcom.js

```diff
@@ -2,22 +2,24 @@
 // Contents of this file are temporal regardless, and logic is somewhat universal regardless.
 const isBrowser = typeof window !== 'undefined';
 
+const dotcomBaseUrl = 'www.mongodb.com/docs-qa';
```

Comment:
hmmm, it's a little confusing having a `dotComBaseUrl` and a `baseUrl` exported - I think making this a little clearer as a const and having the variable name be all uppercase would make it easier to tell at a glance how to use the two 😄 

```suggestion
const DOTCOM_BASE_URL = 'www.mongodb.com/docs-qa';
```

Reply:
Good point! Pushed up a commit with the change